src/lib/score/graphics/widgets/QGraphicsRangeSlider.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+#include <score/graphics/widgets/Constants.hpp>
+
+#include <QGraphicsItem>
+#include <QObject>
+
+#include <score_lib_base_export.h>
+
+#include <verdigris>
+
+///TODO. list in src/plugins/score-plugin-avnd/Crousti/Concepts.hpp#L139
+
+namespace score
+{
+class SCORE_LIB_BASE_EXPORT QGraphicsRangeSlider final
+    : public QObject
+    , public QGraphicsItem
+{
+  W_OBJECT(QGraphicsRangeSlider)
+  Q_INTERFACES(QGraphicsItem)
+  QRectF m_rect{defaultRangeSliderSize};
+
+  double m_start{0.}, m_end{1.};
+  double m_min{0}, m_max{1};
+
+public:
+  bool moving = false;
+
+  explicit QGraphicsRangeSlider(QGraphicsItem* parent);
+
+  void setStart(double start);
+  void setEnd(double end);
+  void setRange(double min, double max);
+
+  void setValue(ossia::vec2f value);
+  ossia::vec2f value() const noexcept;
+  void setExecutionValue(ossia::vec2f) { } // TODO
```

Comment:
it would be nice to have those: they show the current running value in the execution engine. Could be a small line at the bottom for instance